kyo-core/shared/src/main/scala/kyo/Async.scala

```diff
@@ -248,6 +248,47 @@ object Async extends AsyncPlatformSpecific:
     ): A < (Abort[E] & Async & S) =
         race[E, A, S](first +: rest)
 
+    /** Races multiple computations and returns the result of the first to complete. When one computation completes, all other computations
```

Comment:
Can you add some explanation of the difference from the regular `race`? It'd be nice to add a note to `race` as well